File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    let answer = n;
    for (let i = 0; i < control.length; i++) {
        switch (control[i]) {
            case 'w':
                answer += 1;
                break;
            case 'a':
                answer -= 10;
                break;
            case 's':
                answer -= 1;
                break;
            case 'd':
                answer += 10;
                break;
        }
    } 
    return answer;
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.py
def solution(n, control):
    answer = n
    for char in control:
        if char == 'w':
            answer += 1
        elif char == 's':
            answer -= 1
        elif char == 'd':
            answer += 10
        elif char == 'a':
            answer -= 10
    return answer